usr/share/rear/verify/BAREOS/default/040_prepare_environment_vars.sh

```diff
@@ -0,0 +1,24 @@
+# read config vars defined during prep.
+source "$VAR_DIR/bareos.conf"
+
+if [ -z "$BAREOS_CLIENT" ]; then
+    BAREOS_CLIENT="$HOSTNAME-fd"
+    export BAREOS_CLIENT
+fi
+
+if [ -n "$BAREOS_FILESET" ]; then
+    FILESET="fileset=\"$BAREOS_FILESET\""
+    export FILESET
+fi
+
+if [ -n "$BAREOS_RESTORE_JOB" ]; then
+    RESTOREJOB="restorejob=\"$BAREOS_RESTORE_JOB\""
+    export RESTOREJOB
+    RESTOREJOB_AS_JOB="job=\"$BAREOS_RESTORE_JOB\""
+    export RESTOREJOB_AS_JOB
+fi
+
+if [ -n "$BAREOS_RESTORE_CLIENT" ]; then
```

Comment:
If this is for cross-client restore then I would expect it to be set / asked for during `rear recover` and not predetermined during `rear mkrescue` or part of the ReaR configuration.

Reply:
removed again.